src/Nncase.Core/CompileOptions.cs

```diff
@@ -115,6 +115,11 @@ public sealed record CompileOptions
     /// </summary>
     public ShapeBucketOptions ShapeBucketOptions { get; set; } = ShapeBucketOptions.Default;
 
+    /// <summary>
+    /// Gets or sets huggingface options.
+    /// </summary>
+    public HuggingFaceOptions HuggingFaceOptions { get; set; } = HuggingFaceOptions.Default;
```

Comment:
`HuggingFaceOptions` should be an import options.

Reply:
Next month.